code/modules/shuttle/shuttle.dm

```diff
@@ -27,9 +27,13 @@
 	var/hidden = FALSE //are we invisible to shuttle navigation computers?
 
 	//these objects are indestructable
+
+
+/var/my_prohibited_global_var = 3
```

Comment:
?

Reply:
forgot to back out the actual testing changes